client/components/domains/register-domain-step/index.jsx

```diff
@@ -220,7 +221,10 @@ const RegisterDomainStep = React.createClass( {
 	},
 
 	render: function() {
-		const queryObject = getQueryObject( this.props );
+		const queryObject = getQueryObject( this.props ),
+			placeholder = ( this.props.isSignupStep && abtest( 'signupDomainsHeadline' ) === 'updated' )
```

Comment:
I've added the `this.props.isSignupStep` so that users that use domain search in Domain Management in Calypso won't get bucketed into this abtest preemptively (this field is re-used in signup).